docs/source/start.ipynb

```diff
@@ -37,7 +38,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X, y = evalml.demos.load_breast_cancer()"
+    "X, y = evalml.demos.load_fraud(n_rows=1000)"
```

Comment:
You can do `evalml.demos.load_fraud(n_rows=1000, return_pandas=True)` to avoid having to call `.to_dataframe()`